CONTRIBUTING.adoc

```diff
@@ -272,22 +274,12 @@ at the top of the dind-cluster.sh script.
 
 Attempting to start a cluster when one is already running will result
 in an error message from docker indicating that the named containers
-already exist.  To redeploy a cluster after making changes, use the
-'start' and 'stop' or 'restart' commands.  OpenShift is always built
-as part of the dind cluster deployment initiated by 'start' or
-'restart'.
-
-By default the cluster will consist of a master and 2 nodes.  The
-OPENSHIFT_NUM_MINIONS environment variable can be used to override the
-default of 2 nodes.
-
-Containers are torn down on stop and restart, but the root of the
-origin repo is mounted to /data in each container to allow for a
-persistent installation target.
-
-While it is possible to run a dind cluster on any host (not just a
-vagrant VM), it is recommended to consider the warnings at the top of
-the dind-cluster.sh script.
+already exist.  To redeploy a cluster use the 'start' command with the
+'-r' flag to remove an existing cluster.
+
+While it is possible to run a dind cluster directly on a linux host,
+it is recommended to consider the warnings at the top of the
+dind-cluster.sh script.
```

Comment:
I assumer there are some folks for whom disabling selinux and running privileged docker containers directly on their host won't be desirable.  Fair point, though, that there isn't much detail on how they would run a dind cluster otherwise.